ceph.spec.in

```diff
@@ -1437,6 +1437,15 @@ fi
 %endif
 %{_bindir}/ceph-post-file
 %{_tmpfilesdir}/ceph-common.conf
+%if 0%{with python2}
+%dir %{python_sitelib}/ceph
+%{python_sitelib}/ceph
+%{python_sitelib}/ceph-*.egg-info
+%else
+%dir %{python3_sitelib}/ceph
+%{python3_sitelib}/ceph
+%{python3_sitelib}/ceph-*.egg-info
+%endif
```

Comment:
@sebastian-philipp Have you tried this to see if it does what you expect?

Reply:
I have tried it in Sepia, but as there is no Py3 coverage: No, I haven't tried it. But this is exactly how `ceph_volume` is packaged!

Reply:
`%{python3_sitelib}/ceph` is a directory tree containing multiple files and subdirectories? In that case, I believe the line `%dir %{python?_sitelib}/ceph` should be dropped.

Reply:
Both: `%{python3_sitelib}/ceph` and `%{python3_sitelib}/ceph-*.egg-info` are directories. Ok. I'll remove  `%dir %{python3_sitelib}/ceph` then!